src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1353,7 +1355,6 @@ internal virtual HttpResponseMessage GetResponse(HttpClient client, HttpRequestM
                     _cancelToken.Cancel();
                     _cancelToken = null;
 
-                    req.Dispose();
```

Comment:
Why is this removed? It is previous request and we should dispose it.

Reply:
I removed it because I thought `req = GetRequest(currentUri);` on the next line made the dispose superfluous, I can revert

Reply:
I think it is better to explicitly dispose. Please revert.

I have more concerns how we process HttpResponseMessage. It seems we don't dispose. We could investigate this in follow PR.

Reply:
reverted